portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,51 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  //Variable to hold an ArrayList of values to convert to JSON.
+  public ArrayList<String> msg;
```

Comment:
nit:
private List<String> msg;

In java, it's good practice to use the more general List interface when defining a variable. 

And the field probably should be private?

Reply:
ok, just fixed!

Reply:
Not sure if the PR is not updating, but I am still seeing `public ArrayList<String> msg;`

Reply:
Is it fixed now? I just pushed again!

Reply:
Ahh the comment does not update itself, I have to go to code and then change the commit I am looking at.